### DISCUSSION TITLE
A way to create a configuration node to allow the use of Hue bridge IP and key

### DISCUSSION THREAD

**Discussion by toaster-code** [andesse/hue-clip-api.node-red-flows#21] [Ideas]
I know maybe I'm too lazy, but it would be nice to proper integrate the info obtained from yadomi module instead of retyping them into the "HUE Bridge config" node.
Not a real great improvement but I was looking in how to do this and I found that nodes does not allow this kind of information share without a message or an flow or global enviroment variable.
I wonder if its possible to build a configuration node to do it.

**Comment by andesse** [comment]
@toaster-code the V2 prefix can do this. You just need to download the Event_Sender

but thanks for ideas! GitHub lives on ideas :)

and a comment from me! That how we get into parsing again. My idea was to avoid that! I found the effort better to set it up once and DONE. I personally want the flow as clean as possible, with the minimum on code, I want the fastest / quickest responses for my request. Even 200-300ms I can feel the delay already. Light have to be instant on on Button press, so I kept it simply

Greetings,
Andreas

**Reply by toaster-code** [reply]
I confess that I did not try the V2. I started directly with the V4.
that is the problem with version numbers (why bother with something that could be considered "obsolete"?)

Now I understand that they are different "flavors" to do the same thing. Thanks for the feedback.

**Reply by toaster-code** [reply]
Sorry, now I see that V4 uses a V2 prefix...

**Reply by andesse** [reply]
Ok, it’s described in the first scentence of the readme. My approach isn’t using parsing. 
I don’t like processes the probably slowing it down.

V2 is a bit more convince for starters
